form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/web/rest/FormFlowResource.kt

```diff
@@ -60,6 +61,7 @@ class FormFlowResource(
     fun completeStep(
         @PathVariable(name = "instanceId") instanceId: String,
         @PathVariable(name = "stepId") stepId: String,
+        @RequestParam(name = "openNext", defaultValue = "false") openNext: Boolean = false,
```

Comment:
How are we going to handle this scenario:
1. User fills in form
2. User completes the form
3. ???
4. User closes window
5. 10 min later, the user opens window again 
6. ??? 

The 'onOpen' expressions, are they executed at step 3 or step 6? or both?

Reply:
Is that a scenario we have to cover though? Especially for the demo endpoints?

Reply:
Im justwondering if we're all on the same page here. This solution seems to suggest that it behaves as an 'onCreate' rather that an 'onOpen'. Because I think it should happen at both step 3 and step 6

Reply:
For the demo it happens in the same REST call (at least that's how it's written right now). I think for next sprint we want to look at the necessary APIs required to do all actions (completing a step, opening it, creating an instance(?)), so then we can just make the split properly.